### ISSUE TITLE
status not showing

### ISSUE THREAD

**Issue by maxipon** [mdnpascual/ToF-Fishing-Bot#22]
at first, I was using the bot normally. but when I tried to run the bot again today, it doesn't show the status (that's right, not even the NotFishing status, just blank). can you look into this?
here is what it look like:
![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/96283289/774043ec-7a6b-43ac-a92e-862fe432edd4)

**Comment by mdnpascual** [comment]
So it works before and now it stops working? 

I don't know. Did you change monitor or something? forgot to run as admin?

Also this is the old v1, can you try updating to the new version. https://github.com/mdnpascual/ToF-Fishing-Bot/releases/download/2.2.1/ToF_Fishing_Bot_v2_2_1.zip

**Comment by maxipon** [comment]
I tried using ver 2.2.1, it gave the same status, and the bar is not showing (also running as admin)

I only using one monitor until now

and the only things I did make change is this: for the last week, I'm trying to get my Minecraft to run with my NVIDIA graphic cards, so I went into my NVIDIA Control Panel and do this (notices the Preferred graphics processor in the first image)

![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/96283289/3bd62dab-457d-4017-b1cc-7b22db739b7c)

I tried changing it back to auto-select, will see if it fixed or not

I am using a laptop, and here is my spec:
![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/96283289/8d670f77-9aa5-4253-8ec2-4c7cd1fb84fe)
![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/96283289/c1592529-040f-46f1-bc16-228b83d775dc)
![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/96283289/686939d2-c2e1-4950-a52e-15f53891c479)

**Comment by mdnpascual** [comment]
ye, that might be it. If you want to use the high performance nvidia processor, you need to research if:

1) You're specific laptop has a mux switch
2) Disable/turn off the AMD Graphics via the mux switch

This then uses the gtx 1650 directly. What is happening right now that is making the program not work is like this:

Game -> Nvidia 1650 -> AMD Radeon graphics -> Monitor. 

We need to cut off the middleman in the middle (amd radeon) for the program to work.

It was working before because it's like

Game -> AMD Radeon graphics -> Monitor.

If there is a mux switch and you can disable AMD radeon graphics. Program should work

**Comment by mdnpascual** [comment]
Actually, wait, you said you are using a 2nd monitor now? 

1) Make the external monitor as your main monitor

<img width="236" alt="image" src="https://github.com/mdnpascual/ToF-Fishing-Bot/assets/1689277/cf279dd3-da09-4397-9bce-34e02f8dc3f2">


2) Put the game there and re-setup the tool

**Comment by maxipon** [comment]
no, I am still only using one monitor at the time I'm writing this issue. I'm testing if changing the program back to using Integrated will help :D

**Comment by mdnpascual** [comment]
I'm releasing a debug build to specify a GPU to target here: https://github.com/mdnpascual/ToF-Fishing-Bot/releases/download/experimental-gpu2/ToF_Fishing_Bot_GPU_Selector.zip

let me know if selecting a different one fixes yours. If it crashes after clicking start fishing, then that GPU is the wrong one to use. 

![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/1689277/50d2e9d5-22ed-4425-ac3a-3e19bff86042)


Can you also screenshot what you see there.

**Comment by mdnpascual** [comment]
If it still doesnt work, try this fix as well.

Right click desktop -> Display settings -> Graphic settings (bottom most) -> Browse QRSL.exe.
Click options and select high performance

![image](https://github.com/mdnpascual/ToF-Fishing-Bot/assets/1689277/237d7c9c-03d9-4720-840d-c0748b8998d4)
